docs/features.rst

```diff
@@ -104,6 +104,85 @@ Features of the debugger:
   shell code can be run (and debugged) outside of make.
 * Enter debugger at the outset, call it from inside a Makefile, or enter it upon the first error
 
+See  :ref:`debugger <debugger>` for more information on the built-in debugger.
+
+For Developers
+--------------
+
+If you are interested in learning about how GNU Make works, you might find it easier to start out working with this code because it has been simplified as a result of the removal of lesser-used OS's.
+
+In contrast to GNU Make, we don't even attempt to support:
+
+* VMS (whether on VAX_ or the OpenVMS_  variant)
+* DOS (with or without EMX_ and DJGCC_),
+* native MS/Windows,
```

Comment:
Not sure you can call Windows a `lesser-used OS` ;)

Reply:
I meant lesser used as far as GNU Make is concerned. MS/Windows has its own way of doing things, which for the most part doesn't include using GNU Make.